packages/taco/test/conditions/predefined/erc721.test.ts

```diff
@@ -44,7 +44,7 @@ describe('ERC721Balance', () => {
       chain: TEST_CHAIN_ID,
       returnValueTest: {
         comparator: '==',
-        value: '10',
+        value: BigInt(10),
```

Comment:
This test only tests that BigInts were not previously accepted. It's not really testing that a large value would have failed in the past.

Reply:
Yes, exactly. This test is for the new feature added (which is to support bigint).

Reply:
I really meant that based on the commit message of - https://github.com/nucypher/taco-web/pull/637/commits/d8773d2d152942f045f8043ef7d92e2d0cae97da i.e. "modify a test that passes now but would fail before the PR" -> this would have failed because of a type mismatch.

I think that instead, you can add an additional test to confirm the ability to use BigInt instead of modifying an existing one - keeping original tests are good for ensuring regression testing. For new functionality we should add new tests.